docs/runbook/Features.md

```diff
@@ -173,90 +98,168 @@ on the rooftop is selected, which can be difficult to do manually.
+2.  Networks shared by businesses and residential homes where businesses require
+    a higher amount of bandwidth.
+
+The feature works by creating multiple demand sites at specified CN locations.
+For the example in #1 above, this means placing 5 demand sites on the CN on the
+building with 5 units and 3 demand sites on the building with 3 units.
+
+**How to use it**
+
+1. Set `ENABLE_CN_DEMAND` to True
+2. Modify the [User Input Site File](Input_Files.md#user-input-site-file)
+   1. For KML/KMZ input, for each relevant site, add a `number of subscribers`
+      data field and set to the desired value.
+   2. For CSV input, add a `number of subscribers` column and set it to the
+      desired value for each relevant site.
+
+The `number of subscribers` field is only applied to CNs and ignored for DNs
+and POPs. If left blank for a CN, it is assumed to be 1.
+
+## Maximize Common Bandwidth
+
+Maximize Common Bandwidth (MCB) equally distributes bandwidth across all
+connected demand sites during the network optimization steps. When MCB is
+disabled, the total shortage (unsatisfied demand) in the network is minimized
+but there are no guarantees of how that shortage is distributed among clients.
 
-Number of active/connectable candidate/total candidate CN sites.
+If the input demand is too high for the underlyling network, some clients can
+be disconnected in order to improve the bandwidth to other clients. Consider
+the following example. Say you have a single POP connected to 10 CNs.
+For simplicity, assume that this POP has one sector and each of the links from
+the POP to the CNs has a capacity of 1.8 Gbps. If you request that each CN
+receives 200 Mbps of service, it will not be possible to satisfy that amount of
+demand, so there will be (200 * 10 - 1800) = 200 Mbps shortage in the network.
+With MCB disabled, that shortage can be split among all the CNs in different
+ways (unfortunately, how it is distributed is unpredictable). It is possible
+that the planner could provide 200 Mbps of service to 9 of the 10 CNs and leave
+1 of the CNs disconnected. This is a common cause of disconnected sites when
+planning networks.
+
+One way to address this is to simply reduce the demand to 180 Mbps instead of
+200 Mbps. However, it can often be difficult to determine what this value
+should be in more complex networks. With MCB enabled, each of the 10 CNs will
+get 180 Mbps of service because the shortage will be distributed among the CNs
+evenly.
+
+Unfortunately, this feature requires solving a few extra optimization problems
+which might make the overall runtime worse. In many cases, simply adding more
+POPs to the network or reducing the requested demand is a better alternative.
+
+**How to use it**
+
+1. To use the Maximize Common Bandwidth feature, set `MAXIMIZE_COMMON_BANDWIDTH` under
+the `NETWORK_DESIGN` section to True.
+
+## Extend Existing Candidate Graph
+
+Extending an existing candidate graph makes it possible to plan a network that
```

Comment:
Minor: Maybe you can reword this paragraph to make it shorter? An example suggestion:

This feature enables extending an already deployed "base network" with new sites. LOS will be computed between the base network sites and the new sites and between each of the new sites. This new candidate graph includes the base network as a subgraph. The candidate graph is then optimized to generate a network plan.